infra/growth/src/resources/rules.js

```diff
@@ -231,10 +232,18 @@ class CampaignRules {
     const level = ethAddress
       ? await this._calculateLevel(ethAddress, events)
       : undefined
+
+    const allRules = flatMap(level => level.rules, Object.values(this.levels))
     const data = []
     for (let i = 0; i < this.numLevels; i++) {
       data.push(
-        ...(await this.levels[i].export(adapter, ethAddress, events, level))
+        ...(await this.levels[i].export(
+          adapter,
+          ethAddress,
+          events,
+          level,
+          allRules
```

Comment:
Do we really need to pass this additional parameters ? Both Level and BaseRule have a "crules" field that gives access to all the rules. So the rules should be already accessible without this extra parameter.

Reply:
good point! Time to un-mess the method signatures